pkg/apis/resiliency/v1alpha1/types.go

```diff
@@ -51,10 +51,16 @@ type Policies struct {
 }
 
 type Retry struct {
-	Policy      string `json:"policy,omitempty" yaml:"policy,omitempty"`
-	Duration    string `json:"duration,omitempty" yaml:"duration,omitempty"`
-	MaxInterval string `json:"maxInterval,omitempty" yaml:"maxInterval,omitempty"`
-	MaxRetries  *int   `json:"maxRetries,omitempty" yaml:"maxRetries,omitempty"`
+	Policy      string           `json:"policy,omitempty" yaml:"policy,omitempty"`
+	Duration    string           `json:"duration,omitempty" yaml:"duration,omitempty"`
+	MaxInterval string           `json:"maxInterval,omitempty" yaml:"maxInterval,omitempty"`
+	MaxRetries  *int             `json:"maxRetries,omitempty" yaml:"maxRetries,omitempty"`
+	Conditions  *RetryConditions `json:"conditions,omitempty" yaml:"conditions,omitempty"`
+}
+
+type RetryConditions struct {
+	HTTPStatusCodes string `json:"httpStatusCodes,omitempty" yaml:"httpStatusCodes,omitempty"`
+	GRPCStatusCodes string `json:"gRPCStatusCodes,omitempty" yaml:"gRPCStatusCodes,omitempty"`
```

Comment:
Please add API comments. These also appear in the kubectl describe output.

Reply:
Done.